common/src/Assets/Texture.h

```diff
@@ -50,7 +50,12 @@ namespace TrenchBroom {
         };
 
         struct TextureBlendFunc {
-            bool enable;
+            enum class Enable {
+                Default,
```

Comment:
So "No" means we always draw fully opaque, "Yes" means with some kind of blending (depending on the texture alpha channel and the src/dest blend modes) and "Default" means potentially alpha masked (depending on the texture alpha channel)? Maybe add a doc comment

Reply:
It turns out we already have all the information we needed, we just didn't use it correctly. I  have changed the PR to use the available info.